More flexible read.sas7bdat.parso

xiaodaigh opened the pull request:
more flexible read.sas7bdat.parso function by allowing for a custom specification of the read function e.g. allow fread to be used instead of read.csv.

Also added a fread.sas7bdat.parso function

Updated DESCRIPTION and NAMESPACE to take into account data.table

xiaodaigh committed:
Update read.sas7bdat.parso.R

Commit:
more flexible read.sas7bdat.parso function by allowing for a custom specification of the read function e.g. allow fread to be used instead of read.csv.

Also added a fread.sas7bdat.parso

Commit:
Update DESCRIPTION and NAMESPACE to take into account data.table

BioStatMatt commented:
Thanks for this contribution. Would you please also edit the man/read.sas7bdat.parso.Rd file appropriately, and also create a man/fread.sas7bdat.parso.Rd file to document the new function. These are required in order to upload the package to CRAN. Once those are ready, I'll merge your changes.

xiaodaigh committed:
Create fread.sas7bdat.parso

xiaodaigh committed:
Update read.sas7bdat.parso.Rd

xiaodaigh committed:
Rename fread.sas7bdat.parso to fread.sas7bdat.parso.rd

xiaodaigh committed:
Delete fread.sas7bdat.parso.rd

xiaodaigh committed:
Update read.sas7bdat.parso.Rd

added documentation on fread.sas7bdat.parso and updated the description of read.sas7bdat.parso

xiaodaigh commented:
I have updated as requested

BioStatMatt commented:
Thanks. There were are few things that needed fixing before the package would pass "R CMD check" without a WARNING or NOTE. In general, it's good practice to make sure that your changes don't cause errors, warnings, or notes.
